### DISCUSSION TITLE
Hi, Team Rainbow Kit.

### DISCUSSION THREAD

**Discussion by good199x** [rainbow-me/rainbowkit#1698] [Ideas]
My name is Sergei and I am a product manager of [ANKR](https://www.ankr.com/) company. I am responsible for the development of the mRPC product.

I would like to discuss with Rainbow Kit the possibility of collaboration. For example develop some new methods for your libraries and enable users to quickly connect RPC provider like ANKR from the box.

Best regards,
Sergei Ladeishchikov
s.ladeishchikov@ankr.com

**Comment by DanielSinclair** [comment]
@good199x Hi Sergei. That piece of dApp development is more so controlled by Wagmi. With Wagmi v2, developers can drop in any RPC provider as a `transport`, so you can independently distribute a simple guide for that or a package to simplify it even further.

**Comment by flymauntains** [comment]
I building rainbowkit source. At this time, need RAINBOW_PROVIDER_API_KEY, where I get it?

**Reply by DanielSinclair** [reply]
@flymauntains This is no longer required. Give it another try!

**Reply by flymauntains** [reply]
Hi, DainelSinclair, I customzied rainbowkit module. Could u check my custom. I sure you interested my customization. TG:Fly0113
